profile/UserProfile.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>UserProfile</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="UserProfile.css">
+
+  </head>
+  <body>
+    <div class="top_bar">
+      <div class="logo">
+        <a href="index.php" style="color: white; padding-left: 5px; font-family: 'Titillium Web', sans-serif; font-size:25;">Logo Here</a>
```

Comment:
Extract CSS to file. Font family for this project is Monserrat (available at fonts.google.com).

Reply:
Do you mean that for all text in this project?